debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -77,7 +77,14 @@ private PostgresReplicationConnection(Configuration config,
 
         try {
             initReplicationSlot();
-        } catch (SQLException e) {
+        }
+        catch (SQLException e) {
```

Comment:
`initReplicationSlot()` method can also throw `IllegalStateException`:
```
if (!rs.next()) {
    throw new IllegalStateException("The DB connection is not a valid replication connection");
}
```

So we should catch `Throwable`, like this:
```
} catch (Throwable e) {
    // If we don't complete initialization, then Kafka Connect will not attempt to stop the connector.
    // So if we run into a problem, we have to stop ourselves ...
    try {
        stop();
    } catch (Throwable s) {
        // Log, but don't propagate ...
        logger.error("Failed to initialize the replication slot (see other exception), but got this error while cleaning up", s);
    }
    if (e instanceof ConnectException) {
        throw (ConnectException) e;
    }
    throw new ConnectException("Cannot create replication connection", e);
} 
```

`initReplicationSlot()` method already catches `SQLException`, so this code should be removed:
```
} catch (SQLException e) {
    throw new JdbcConnectionException(e);
}
```